assets/component-mega-menu.css

```diff
@@ -32,7 +32,9 @@
 
 .mega-menu[open] .mega-menu__content {
```

Comment:
It's needed for shadows that could be on top of the header above the menu. 
https://github.com/Shopify/dawn/pull/1316/files#r811466381
But with the [changes from that older PR that we merged](https://github.com/Shopify/dawn/pull/1912/files) it's actually coming up on top now anyway... 

Ill look into it a bit more when I have time 👍

Reply:
So since the purpose of the z-index here was to prevent the shadow to overlap, and it's not doing it right now due to the other changes I mentioned in my prior comment, I think we could just remove the `z-index: -1;` for now. And revisit how we apply shadows on popups in a follow up. 
What do you think ?
This way we fix the issue at end which I think is worse than shadow overlapping the menu above it.